keyboards/tetris/rules.mk

```diff
@@ -55,4 +55,6 @@ BACKLIGHT_ENABLE ?= no  # Enable keyboard backlight functionality
 AUDIO_ENABLE ?= yes
 RGBLIGHT_ENABLE ?= yes
 TAP_DANCE_ENABLE = yes
```

Comment:
Tap Dance should be disabled by default, and should only be enabled for keymaps that need it. 

This is why it's erroring out in Travis.

Reply:
So disable Tap Dance by

1. change `yes` to `no` here
2. add a `keymaps/default/config.h` file containing `#define TAPPING_TERM 200`
  - The default keymap uses tap dance

Is that correct?